### DISCUSSION TITLE
Account blocked, I need help

### DISCUSSION THREAD

**Discussion by Awesome-GJK** [community/community#49678] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This Monday, I found out that my github account was banned, but I didn't receive any email reminder. Since Monday, I have filed complaints in github support every day, but there is no response, no official contact me.

Since I used github, I have never done any illegal operations, strictly abide by the github account usage rules, I don't know why the official banned me, I don't understand!

I hope the official will contact me as soon as possible to help me unblock my account, thanks.

![image](https://user-images.githubusercontent.com/127288468/224257840-d3fce543-3b69-4ea0-a5b8-beb7b131a798.png)
My original account: https://github.com/GJKGJKGJK
my statement：
![image](https://user-images.githubusercontent.com/127288468/224258071-5873d877-2e53-45d7-841f-c8ab691215e4.png)

**Comment by tmpyc0304** [comment]
Are the support team deal with your account? My account was suspended for a month but haven't get any receive either.

**Reply by Awesome-GJK** [reply]
No processing, my account has been suspended for more than a month, I submitted a complaint, but no official person contacted me or resolved my account issue
